### ISSUE TITLE
Code for each chapter

### ISSUE THREAD

**Issue by bluee** [ProgrammingAspNetMvcBook/CodeExamples#2]
Usually a book about programming have code that is relevant for each chapter. But with this book I only see one final code - EBuy which doesn't even compile. Do you have plans for adding source code that is relevant to each chapter?

**Comment by mardsystems** [comment]
yes, where are all examples????
greatful!

**Comment by jchadwick** [comment]
Sorry, we are still working on completing this.  It will be done within the next few weeks.

**Comment by ghost** [comment]
Hi, Just wanted to add that I would also find this really useful.  It's a great book so far, but I've already got a few problems with the website I'm building and I'm only at chapter 5!

I'll keep checking back for more code samples.

Thanks.

**Comment by minuz** [comment]
Hi, so what happened with the development of the code examples? The last post is from 5 months ago and states that would have been done within a few weeks. Is this project down??
I'm reading the book and it would be really useful to see working examples as an extension of the learning process.

Thanks xD

**Comment by jchadwick** [comment]
Sorry everyone - we all got bogged down, but we are actively working to get the code examples online ASAP.  Please stay tuned.

**Comment by GTres** [comment]
Hi, 
One error:

Error   1   'Ebuy.Website.Models.AuctionViewModel' does not contain a definition for 'Key' and no extension method 'Key' accepting a first argument of type 'Ebuy.Website.Models.AuctionViewModel' could be found (are you missing a using directive or an assembly reference?) c:\Users\guillo\Documents\CodeExamples-master\CodeExamples-master\Ebuy.Website\Views\Search\Search.cshtml   68  Ebuy.Website

**Comment by Lionheart831** [comment]
Still waiting on the examples.... :.( :-1:

**Comment by Notso** [comment]
Yeah authors, come on. I am struggling to completely understand all of Chapter 5, and in order to understand it better, I was hoping to come here and get a working version of the code at the end of Ch. 5, but if cannot even get a working version (as comments above indicate), I am dead in the water; this book becomes useless to me. Do I need to buy another book, or are you authors going to get this done so that there really are code examples for "code throughout this book" as promised in the book?

**Comment by moojjoo** [comment]
Wow six months, better yet a year and this has not been answered.  I will research and provide an answer.

**Comment by moojjoo** [comment]
Line 68 simply remove the .key from item.key

var auctionUrl = Url.Action("Auction", "Auctions", new { id = item });

Now select the Ebuy.Website and build.  Should fix it.  Did for me.
